010-server-side-rendering.md

```diff
@@ -0,0 +1,205 @@
+# Server-Side Rendering
+
+We should use [Next.js](https://nextjs.org) and [Heroku](https://www.heroku.com) to power server-side rendering for Phoenix.
+
+**Discussion:** https://github.com/DoSomething/rfcs/pull/10
+
+## Problem
+
+Our front-end application, Phoenix, is not as scalable, simple, or performant as we want:
```

Comment:
Is anchor linking enough of a problem to make this list? I feel like that was a really nice win when we introduced linking to specific blocks on campaign pages etc. which we lost with dynamically loaded blocks.

Reply:
Oh, good call! I think that definitely belongs here.

Reply:
Added this in df9f5f0.